### DISCUSSION TITLE
Massive copilot blocking fails in copilot chats

### DISCUSSION THREAD

**Discussion by ostermann-felix** [community/community#77728] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have a big problem with copilot chats. I use chats like it should be used but I am often get this message: 

"Sorry, the response matched public code so it was blocked. Please rephrase your prompt. [Learn more](vscode-file://vscode-app/f:/Users/Felix/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html)."

The problem is, that I do basic nextjs front end stuff and i already used the link and activated the presented solution in the settings. It still blocks be out from real basic codesnippets that comes up. Like this Github Copilot is unusable for me. I get those messages with english and german prompts. I need help. What could I do?

PS: I testet the copilt chat with the suggested questions of copilot itself and it blocks its own questions with the same message.

**Comment by Akash1134** [comment]
Hi @ostermann-felix ,

We're sorry about the trouble caused. This issue has been reported by users since Nov 20th and is currently being reviewed by our engineers.

Some users are able to get past the error by writing /clear in chat prompt and restarting their IDE. However, note that is only a unverified workaround. 

Here's the [primary issue](https://github.com/microsoft/vscode-copilot-release/issues/583) where our engineers are gathering all the reports. 

Thanks!

**Comment by dieghard** [comment]
Sorry, the response matched public code so it was blocked. Please rephrase your prompt. [Learn more] tengo todo bien configurado y Copilot sigue fallando :S
